rbac/management/relation_replicator/bootstrapped_tenant_check.py

```diff
@@ -0,0 +1,168 @@
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""RelationReplicator which check relations on Relations API."""
+
+
+import logging
+
+from django.conf import settings
+from google.protobuf import json_format
+from grpc import RpcError
+from internal.jwt_utils import JWTManager, JWTProvider
+from kessel.relations.v1beta1 import check_pb2
+from kessel.relations.v1beta1 import check_pb2_grpc
+from kessel.relations.v1beta1 import common_pb2
+from management.cache import JWTCache
+from management.utils import create_client_channel
+
+jwt_cache = JWTCache()
+jwt_provider = JWTProvider()
+jwt_manager = JWTManager(jwt_provider, jwt_cache)
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class RelationsApiTenantChecker:
```

Comment:
is it possible to leverage RelationsApiRelationChecker ? 

RelationsApiTenantChecker is currently couple with tenant mapping - idea is just to send relations to `RelationsApiRelationChecker`